packages/@sfx/cache-driver-plugin/src/index.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I needed to add this `eslint-disable` line because there is a single export below and, by our convention, it is not a default export.